dumoRecordsHandler/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.14)
+project(dumpRecordsHandler)
+
+set(CMAKE_CXX_STANDARD 17)
+
+add_executable(dumpRecordsHandler
+        main.cpp
+        Interfaces/IPerson.h
```

Comment:
Please do not mix headers with cpp. In fact, you shouldn't add headers to executable.